core/src/components/alert/test/translucent/index.html

```diff
@@ -27,12 +27,10 @@
       </ion-header>
 
       <ion-content class="ion-padding">
-        <ion-button expand="block" onclick="presentAlert()">Alert</ion-button>
-        <ion-button expand="block" color="secondary" onclick="presentAlertLongMessage()">Alert Long Message</ion-button>
-        <ion-button expand="block" color="danger" onclick="presentAlertMultipleButtons()"
-          >Multiple Buttons (>2)</ion-button
-        >
-        <ion-button expand="block" color="light" onclick="presentAlertNoMessage()">Alert No Message</ion-button>
+        <button expand="block" class="blue" onclick="presentAlert()">Alert</button>
+        <button expand="block" class="light-blue" onclick="presentAlertLongMessage()">Alert Long Message</button>
+        <button expand="block" class="red" onclick="presentAlertMultipleButtons()">Multiple Buttons (>2)</button>
+        <button expand="block" class="light" onclick="presentAlertNoMessage()">Alert No Message</button>
```

Comment:
`expand="block"` is an `ion-button` API, so it should be removed if we are going to use `button` elements. (Same for all other instances of this)